solutions/security/get-started/spaces-defend-faq.md

```diff
@@ -0,0 +1,246 @@
+**How do spaces impact the visibility of endpoints in the {{security-app}}?**
+
+The list of endpoints that you see depends on your current space. Only endpoints associated with policies in the space you're working in will appear.
+
+
+**How do spaces impact the visibility of {{elastic-defend}} integration policies in {{elastic-sec}}?**
+
+The **Policies** list displays only the policies associated with your current space. The endpoint agent count for each policy includes only the endpoints within that space.
+
+
+## Endpoint artifacts [spaces-security-faq-endpoint-artifacts]
+
+**What are endpoint artifacts?**
+
+Endpoint artifacts are the various objects that can be associated with endpoints and {{elastic-defend}} policies. These include trusted applications, event filters, host isolation exceptions, and blocklist items. Artifacts can be global (shared across all policies) or per-policy (specific to individual policies). Per-policy configuration of artifacts is available only with an Enterprise license.
+
+
+**How do global artifacts work across spaces?**
+
+Global artifacts are space agnostic and thus eppear in all spaces.
+
+
+**How do policy-specific artifacts work across spaces?**
+
+Users can assign artifacts to any policies they have access to within their assigned space.
+
+When an artifact entry is created within a space, it is owned by that space. To edit or delete the artifact, you must either be in the owning space or have **Global artifact management** privileges. 
+
+
+**What happens if my policy uses an artifact owned by a space I don't have access to?**
+
+When viewing a policy, you will still see all artifacts associated with it - regardless of which space they were created in. However, artifacts viewed outside of their owning space will appear as read-only.
+
+If an artifact is associated with a policy that isn't visible in the current space, only the policy's UUID will appear in the "Applied to the following policies" pop-up. For policies accessible within the space, the full policy name will appear.
+
+
+**Why is an endpoint artifact marked as “read-only”?**
+
+An artifact may appear as read-only if:
+- It is a global artifact, and you do not have **Global artifact management** privileges.
+- The artifact was created in a different space.
+
+In these situations, editing may be disabled, and tooltips will provide additional context.
+
+
+**How can I tell which space “owns” a per-policy artifact?**
+
+This information is not currently visible in the Kibana UI. It is, however, available on each artifact record returned by the API under the `tag` field. It will includes a value that corresponds to the owner space's ID in the format of `ownerSpaceId:<space_id_here>`, for example: `ownerSpaceId:default`. By default, each artifact will have at least one such tag, but multiple tags are also supported and cause a per-policy artifact to be managed by multiple spaces.
```

Comment:
small correction:

```suggestion
This information is not currently visible in the Kibana UI. It is, however, available on each artifact record returned by the API under the `tag` field. It will include a value that corresponds to the owner space's ID in the format of `ownerSpaceId:<space_id_here>`, for example: `ownerSpaceId:default`. By default, each artifact will have at least one such tag, but multiple tags are also supported and cause a per-policy artifact to be managed by multiple spaces.
```